internal/config/config.go

```diff
@@ -96,6 +100,10 @@ var DefaultConfig Config = Config{
 	KafkaGroupID:            "config-manager",
 	KafkaInventoryTopic:     "platform.inventory.events",
 	KafkaSystemProfileTopic: "platform.inventory.system-profile",
+	KafkaUsername: 			 os.Getenv("KAFKA_USER_NAME"),
```

Comment:
These should be hard-coded default values, not environment variable values (the lookup from environment variable values happens later).